tests/utils.js

```diff
@@ -133,6 +133,17 @@ const updateSettings = updates => {
       });
     });
 };
+const updatePermissions = async (roles, addPermissions, removePermissions = []) => {
+  const settings = await module.exports.getSettings();
+  if(!('permissions' in settings)){ 
+    settings.permissions = [];
+  }
+  addPermissions.forEach(permission => settings.permissions[permission].push(...roles));
```

Comment:
Please null check in case the permission is missing from the config for some reason.
```
addPermissions.forEach(permission => {
    if (!settings.permissions[permission]) {
      settings.permissions[permission] = [];
    }
    settings.permissions[permission].push(...roles));
});
```